src/KeyringController.ts

```diff
@@ -30,20 +36,21 @@ import type {
   GenericEncryptor,
   ExportableKeyEncryptor,
 } from './types';
+import { throwError } from './utils';
 
 const defaultKeyringBuilders = [
   keyringBuilderFactory(SimpleKeyring),
   keyringBuilderFactory(HDKeyring),
 ];
 
 class KeyringController extends EventEmitter {
-  keyringBuilders: { (): Keyring<Json>; type: string }[];
+  keyringBuilders: { (): EthKeyring<Json>; type: string }[];
```

Comment:
Since there's a type change from `Keyring` to `EthKeyring` it'd be good to indicate the breaking change in the PR description

Reply:
Indeed, good point.